src/Mod/Ship/shipAreasCurve/PlotAux.py

```diff
@@ -34,7 +34,7 @@ def __init__(self, x, y, disp, xcb, ship):
         @param x X coordinates.
         @param y Transversal computed areas.
         @param disp Ship displacement.
-        @param xcb Bouyancy center length.
```

Comment:
You can see in the parameter list to the `__init__()` method that `xcb` is the parameter name and that `Bouyancy` is part of the description.  Also, the first word after @param is the parameter name and the rest is the description.